src/tests/JIT/Regression/JitBlue/Runtime_87597/Runtime_87597.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="1" />
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredPGO" Value="1" />
```

Comment:
I think you should remove these, or you should specify that the test requires process isolation as well.

Reply:
Thanks, will remove and run pgo pipelines (TC and PGO are enabled there)